tools/generate_hotkey.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
+
+
+general = []
+msg_actions = []
+msg_compose = []
+navigation = []
+searching = []
+stream_list = []
+
+CATEGORY = {
+    'general': general,
+    'msg_actions': msg_actions,
+    'msg_compose': msg_compose,
+    'navigation': navigation,
+    'searching': searching,
+    'stream_list': stream_list,
+}
+
+for key, item in KEY_BINDINGS.items():
+    CATEGORY[item['key_category']].append((item['help_text'], item['keys']))
```

Comment:
Would this be clearer as a dict comprehension?

I'll need to reread again later, but is CATEGORY a constant? If not, it doesn't need to be capitalized.

Reply:
I have tried using dict comprehension in two different ways. I have left one commented in the code. Please let me know which one would be preferred.